aioshelly/wsrpc.py

```diff
@@ -88,7 +88,7 @@ async def connect(self, aiohttp_session: aiohttp.ClientSession) -> None:
         _LOGGER.debug("Trying to connect to device at %s", self._ip_address)
         try:
             self._client = await aiohttp_session.ws_connect(
-                f"http://{self._ip_address}/rpc"
+                f"http://{self._ip_address}/rpc", heartbeat=WS_HEARTBEAT
```

Comment:
If you do this, you can drop the manual added heartbeat with timeout

Reply:
162-164 + 185

Reply:
Removed